app/tests/compilers/css.js

```diff
@@ -0,0 +1,4 @@
+function noop() { return null; }
+
+require.extensions['.css'] = noop;
```

Comment:
I don't believe we need a compiler for jsx. Mainly because we require babel-register in setup. My understanding is that it will babelify modules as we require them. The extensions we need to ignore are the ones we babel will not understand, e.g. css.